src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx

```diff
@@ -0,0 +1,27 @@
+function ReviewSummary ({reviews}) {
+
+    let reviewCount = {reviews}.reviews.length
+    let reviewArray = {reviews}.reviews
```

Comment:
No need to do {reviews} - you can just do reviews.length

Reply:
You also probs don't need to make reviewArray, just use `reviews` in your foreach

Reply:
Thanks - resolved both!